ccdproc/combiner.py

```diff
@@ -328,7 +330,10 @@ def average_combine(self, scale_func=ma.average, scale_to=None):
            scale_to : float, optional
                Scaling factor used in the average combined image. If given,
                it overrides ``CCDData.scaling``. Defaults to None.
-
+               
+           uncertainty_func: function, optional
+                Function to calculate uncertainty. Defaults to ma.std
```

Comment:
Maybe you could link to the function since ``std`` is in my opinion not unambiguous in it's meaning: \`numpy.ma.std\`